### PR TITLE
The Docs Always Need Fixing

### PR DESCRIPTION
**Pull request by jaydenmilne**
Various documentation fixes

### CODE REVIEW

File: docs/packages/docs/scenarios.rst [hunk 1]
```diff
@@ -128,13 +134,52 @@ Sensor Objects
...
       "configuration": {
-         
+
       }
    }
 
-The only keys that are required in a sensor object is ``"sensor_type"``, the 
+Sensors have a couple options for placement.
+
+1. **Provide a socket name**
```
Comment (jaydenmilne) [inline docs/packages/docs/scenarios.rst]: Max this section is correct, right?

File: src/holodeck/agents.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ class ControlSchemes:
 
     # HandAgent Control Schemes
     HAND_AGENT_MAX_TORQUES = 0
-    HAND_AGENT_MAX_TORQUES = 1    
+    HAND_AGENT_MAX_SCALED_TORQUES = 1
```
Comment (daniekpo) [inline src/holodeck/agents.py]: Why this change?

Comment (jaydenmilne) [reply src/holodeck/agents.py]: The same name was repeated twice

File: docs/packages/docs/scenarios.rst (hunk 1 above)
Comment (jaydenmilne) [reply docs/packages/docs/scenarios.rst]: Paging @MaxRobinsonTheGreat

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### Prerequisites
+
+`pip install sphinx autodocsumm sphinx_rtd_theme doc8`
+
+### Building
+
+From this directory,
+```console
+jayden@NAGA:~/dev/holodeck/docs$ make clean && make html
```
Comment (MaxRobinsonTheGreat) [inline docs/README.md]: Do you need the `jayden@NAGA` part?

File: docs/changelog/changelog.rst [hunk 4]
```diff
@@ -21,20 +22,22 @@ Highlights
...
 
 Bug Fixes
 ~~~~~~~~~
-
+- Fixed UAV blades rotating incorrectly (thanks @sethmnielsen!)
+  (`#331 <https://github.com/BYU-PCCL/holodeck/issues/331>`_)
 
 Known Issues
 ~~~~~~~~~~~~
-- The TurtleAgent does not move consistently between Linux and Windows.
```
Comment (MaxRobinsonTheGreat) [inline docs/changelog/changelog.rst]: Why was this removed? We still have this issue. Are we planning on fixing this for 0.3.1?

File: docs/packages/docs/scenarios.rst [hunk 5]
```diff
@@ -6,39 +6,45 @@ Scenarios
...
 Scenario File Format
 --------------------
 
-Scenario files are distributed in packages (see :ref:`package-contents`), and
-must be named ``{WorldName}-{PackageName}.json``. By default they are stored
-in the ``worlds/{PackageName}``, but they can be loaded externally as well.
+Scenario ``.json`` files are distributed in packages (see
+:ref:`package-contents`), and must be named
+``{WorldName}-{ScenarioName}.json``. By default they are stored in the
+``worlds/{PackageName}`` directory, but they can be loaded externally as a
```
Comment (MaxRobinsonTheGreat) [inline docs/packages/docs/scenarios.rst]: The word "externally" seems confusing here. I think it should just be removed.

File: docs/packages/docs/scenarios.rst [hunk 6]
```diff
@@ -76,47 +81,48 @@ Agent objects
...
 
 ====================== ========================
 Agent Type             String in agent_type
 ====================== ========================
 :ref:`android-agent`    ``AndroidAgent``
+:ref:`hand-agent`       ``HandAgent``
 :ref:`turtle-agent`     ``TurtleAgent``
 :ref:`nav-agent`        ``NavAgent``
 :ref:`sphere-agent`     ``SphereAgent``
 :ref:`turtle-agent`     ``TurtleAgent``
```
Comment (MaxRobinsonTheGreat) [inline docs/packages/docs/scenarios.rst]: UavAgent?

File: docs/packages/docs/scenarios.rst (hunk 1 above)
Comment (MaxRobinsonTheGreat) [reply docs/packages/docs/scenarios.rst]: yes

File: docs/agents/hand-agent.rst [hunk 7]
```diff
@@ -18,39 +18,42 @@ Control Schemes
...
 
+  The last coordinates allow the HandAgent to float around.
+
+
 .. _`hand-joints`:
 
 HandAgent Joints
 ----------------
 The control scheme for the HandAgent and the
-:class:`~holodeck.sensors.JointRotationSensor` use a 94 length vector refer
```
Comment (MaxRobinsonTheGreat) [inline docs/agents/hand-agent.rst]: Any particular reason for the removal of tailing spaces? I approve of it but am curious if it affects formatting.

Comment (jaydenmilne) [reply docs/agents/hand-agent.rst]: The linter yells at you when you have trailing whitespace. There is a VSCode command to trim them.

File: docs/changelog/changelog.rst (hunk 4 above)
Comment (jaydenmilne) [reply docs/changelog/changelog.rst]: I didn't want to have a long, ever growing list of bugs in this section. It's a known defect with 0.3.0 and will carry on until we fix it

File: docs/README.md (hunk 3 above)
Comment (jaydenmilne) [reply docs/README.md]: It's my calling card
